src/components/custom-widget/customWidgetModelBinder.publish.ts

```diff
@@ -3,15 +3,12 @@ import { Contract } from "@paperbits/common";
 import { CustomWidgetModel } from "./customWidgetModel";
 import { CustomWidgetContract } from "./customWidgetContract";
 import { widgetName } from "./constants";
+import { customWidgetPrefixName, customWidgetRemovePrefixName } from "./ko/utils";
 
 export class CustomWidgetModelBinder implements IModelBinder<CustomWidgetModel> {
-    public canHandleModel(model: unknown): boolean {
-        return model instanceof CustomWidgetModel;
-    }
-
```

Comment:
@azaslonov according to Igor, removing canHandleModel fncs won't break anything. I just want to make sure with you that it's okay.

Reply:
Yup, this was needed in previous model.